docker/frontend.dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:bionic
+
+ENV LANG C.UTF-8
+
+# install latest python and nodejs
+RUN apt-get update && \
+    apt-get install -y \
+      nodejs \
+      npm \
+      gettext \
+      git \
+      sudo \
+      build-essential \
+      python3 \
+      python3-dev \
+      python3-pip \
+      libxml2-dev \
+      libxslt1-dev \
+      zlib1g-dev
+
+RUN bash -c "pip3 install pip setuptools --upgrade --force-reinstall"
+RUN bash -c "pip3 install virtualenv"
```

Comment:
Is there a reason to use virtualenv?

Reply:
Python requirements are installed into a virtualenv, they are less predictable if installed into a system env - things are more likely to break with time.

Reply:
I wouldn't use a virtual environment either.